src/VisualStudio/Core/Def/Implementation/Versions/SemanticVersionTrackingService.cs

```diff
@@ -1,229 +0,0 @@
-﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
-
-using System;
-using System.Composition;
-using System.IO;
-using System.Runtime.CompilerServices;
-using System.Threading;
-using System.Threading.Tasks;
-using Microsoft.CodeAnalysis;
-using Microsoft.CodeAnalysis.Host;
-using Microsoft.CodeAnalysis.Host.Mef;
-using Microsoft.CodeAnalysis.Shared.Extensions;
-using Microsoft.CodeAnalysis.Shared.Utilities;
-using Microsoft.CodeAnalysis.Versions;
-using Roslyn.Utilities;
-
-namespace Microsoft.VisualStudio.LanguageServices.Implementation.Versions
-{
-    /// <summary>
-    /// this service tracks semantic version changes as solution changes and provide a way to get back to initial project/semantic version
-    /// pairs at the solution load 
-    /// </summary>
-    [ExportWorkspaceService(typeof(ISemanticVersionTrackingService), ServiceLayer.Host), Shared]
-    internal class SemanticVersionTrackingService : ISemanticVersionTrackingService
```

Comment:
long time ago, we attempt to save semantic version across VS session, this is code for such attempt. but long time ago, we gave up on that, and disabled tracking but left the code in. basically these are dead code. I removed all those. it should be isolated in each own commit.